### ISSUE TITLE
Documentation for understanding/following the code

### ISSUE THREAD

**Issue by PuneetKohli** [sniklaus/3d-ken-burns#17]
Hi there, 

Could you please consider adding documentation on reading the code, and which files to follow for what?

I would also consider adding function-level comments.

**Comment by sniklaus** [comment]
I agree that it would be useful to have a better documentation, I will add some comments once I have some spare time. I am afraid that this may take a while though, my apologies.

**Comment by PuneetKohli** [comment]
No worries, I can also work with you to do something. We're really interested in evaluating the code/model. 

Do you also plan to release the dataset at some point?

**Comment by sniklaus** [comment]
Yes, I am planning to release the dataset since I acknowledge its usefulness for the research community. I have yet to start the necessary paperwork though and unfortunately have no estimate of when I will be able to release it.

**Comment by PuneetKohli** [comment]
Curious about the technique for generating the data in Unreal. You did mention in the paper that it was photorealistic scenes, but I'm curious if there are certains caveats or "know-how" developed during the capture of the dataset (for example, placement of camera, etc) which would generally be useful for someone to create a similar dataset?

**Comment by sniklaus** [comment]
It took me three months to create the dataset, so there is quite a lot of know-how that I acquired during this time. I have also gone through several iterations of the dataset since the 3D Ken Burns paper and should take the time at some point to summarize it in a technical report.

**Comment by PuneetKohli** [comment]
Interesting. I'm working on capturing a dataset in Unreal using similar techniques (from what I could gather from the Ken Burns paper). If you have a short summary or some major points of reference to share here, that'd be great!
